position/position.gno

```diff
@@ -0,0 +1,1174 @@
+package position
+
+import (
+	"encoding/base64"
+	"std"
+	"strconv"
+
+	"gno.land/p/demo/avl"
+	"gno.land/p/demo/ufmt"
+	"gno.land/r/demo/wugnot"
+	"gno.land/r/gnoswap/v1/gnft"
+
+	u256 "gno.land/p/gnoswap/uint256"
+
+	"gno.land/r/gnoswap/v1/common"
+	"gno.land/r/gnoswap/v1/consts"
+
+	en "gno.land/r/gnoswap/v1/emission"
+	pl "gno.land/r/gnoswap/v1/pool"
+)
+
+const (
+	ZERO_LIQUIDITY_FOR_FEE_COLLECTION = "0"
+)
+
+var (
+	positions = avl.NewTree() // tokenId[uint64] -> Position
```

Comment:
```suggestion
	positions = avl.NewTree() // positionID[uint64] -> Position
```

TokenID is confusing and implies a mapping of denom to number, while it should represent position to number.